rollup.config.js

```diff
@@ -0,0 +1,61 @@
+import babel from 'rollup-plugin-babel';
+import commonjs from 'rollup-plugin-commonjs';
+import replace from 'rollup-plugin-replace';
+import resolve from 'rollup-plugin-node-resolve';
+import uglify from 'rollup-plugin-uglify';
+
+import pkg from './package.json';
+
+function distBuild(options = {}) {
+    return {
+        input: 'src/index.js',
+        output: {
+            file: `dist/${options.filename}`,
+            format: options.format,
+            name: 'animated',
+            sourcemap: options.sourcemap,
+        },
+        plugins: [
+            babel({
+                exclude: ['node_modules/**'],
+            }),
+            replace({
+                'process.env.NODE_ENV': JSON.stringify('production'),
+            }),
+            resolve({
+                browser: true,
+            }), // so rollup can find node modules
+            commonjs(), // so rollup can convert node modules to ESM if needed
+            options.minify && uglify(),
+        ]
+    };
+}
+
+function standardBuilds(filename) {
+    return {
+        input: `src/${filename}.js`,
+        external: [
+            ...Object.keys(pkg.dependencies),
+            ...Object.keys(pkg.peerDependencies)
+        ],
+        output: [
+            { file: `lib/${filename}.js`, format: 'cjs' },
+            { file: `lib/${filename}.mjs`, format: 'es' },
+        ],
+        plugins: [
+            babel({
+                exclude: ['node_modules/**'],
+            }),
+            commonjs(), // so rollup can convert node modules to ESM if needed
+        ]
+    };
+}
+
+export default [
+    distBuild({ filename: 'animated.umd.js', format: 'umd', sourcemap: true, minify: false }),
+    distBuild({ filename: 'animated.umd.min.js', format: 'umd', sourcemap: true, minify: true }),
```

Comment:
Are you already publishing UMD builds? If not, I'm not really sure how much value it adds to start publishing them now, especially if nobody is asking for them. I'd probably leave this out for now.